superbench/benchmarks/micro_benchmarks/dist_inference.py

```diff
@@ -201,6 +204,12 @@ def add_parser_arguments(self):
         """Add the specified arguments."""
         super().add_parser_arguments()
 
+        self._parser.add_argument(
+            '--use_cpp_impl',
+            action='store_true',
+            required=False,
+            help='Whether to use cpp-based implementation.',
```

Comment:
Whether to use pytorch implementation. If not, cpp implementation will be used.

Reply:
Fixed